openmcyclus/DepleteReactor.py

```diff
@@ -161,6 +162,7 @@ def tick(self):
         fuel is loaded
         '''
         if self.retired():
+            # print("time:", self.context.time, "retired")
```

Comment:
Should these commented lines be removed?
```suggestion
```

Reply:
Each of the print statements (commented out and not) are meant for debugging help during development. I have not been removing them so that I can be consistent with where in the simulations I am getting information from. I plan to remove them once OpenMCyclus is in a non-development state.